kpops/components/base_components/base_defaults_component.py

```diff
@@ -64,39 +65,35 @@ class BaseDefaultsComponent(DescConfigModel, ABC):
         description=describe_attr("enrich", __doc__),
         exclude=True,
     )
-    config: SkipJsonSchema[KpopsConfig] = Field(
-        default=...,
-        description=describe_attr("config", __doc__),
-        exclude=True,
-    )
-    handlers: SkipJsonSchema[ComponentHandlers] = Field(
-        default=...,
-        description=describe_attr("handlers", __doc__),
-        exclude=True,
-    )
+    _config: KpopsConfig = PrivateAttr()
```

Comment:
necessary because otherwise it clashes with the newly renamed KafkaConnector config field